src/Microsoft.CodeQuality.Analyzers/Microsoft.CodeQuality.Analyzers.md

```diff
@@ -490,6 +490,14 @@ Help: [Source](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Micros
 
 Maintainability
 ----------------------------------
+### CA1507: Use nameof to express symbol names ###
+
+Using nameof helps keep your code valid when refactoring.
+
+Category: Usage
+
+Severity: Warning
```

Comment:
Do we not have a link for the analyzer?

Reply:
@chborl Can you please work with @gewarren to ensure that we get a docs page for this new analyzer? We should mention that this analyzer/code analysis rule shipped first with version 2.6.0 of FxCop analyzers.